app/uk/gov/hmrc/cgtpropertydisposals/models/LocalDateUtils.scala

```diff
@@ -21,6 +21,41 @@ import java.time.LocalDate
 import cats.Order
 
 object LocalDateUtils {
+  val monthNames: Map[Int, String] = Map(
+    1  -> "January",
+    2  -> "February",
+    3  -> "March",
+    4  -> "April",
+    5  -> "May",
+    6  -> "June",
+    7  -> "July",
+    8  -> "August",
+    9  -> "September",
+    10 -> "October",
+    11 -> "November",
+    12 -> "December"
+  )
+
+  val monthNamesInShort: Map[Int, String] = Map(
```

Comment:
looks like we don't need the short version of the months

Reply:
Done.